.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch sources
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: npm install
+
```

Comment:
@michelkaporin, nice to see ✅ in PR checks :).
I recommend to add caching for NPM dependencies, to speed up the check: https://github.com/actions/cache/blob/main/examples.md#node---npm, wdyt?

Reply:
> Note: It is not recommended to cache node_modules, as it can break across Node versions and won't work with npm ci

from your link, which is why I didn't add it. Added it now anyway, as it might provide some speed-up for NPM package resolution.